cloud/google/gce_img.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/python
+# Copyright 2015 Google Inc. All Rights Reserved.
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+"""An Ansible module to utilize GCE image resources."""
+
+import sys
+
+# import module snippets
+from ansible.module_utils.basic import *
+from ansible.module_utils.gce import *
+
+try:
+  from libcloud.compute.types import Provider
+  from libcloud.compute.providers import get_driver
+  from libcloud.common.google import GoogleBaseError
+  from libcloud.common.google import ResourceNotFoundError
+  _ = Provider.GCE
+except ImportError:
+  print('failed=True '
```

Comment:
This type of "error" here will not be interpreted by ansible correctly any more since "Baby JSON" support was removed in 1.8.

The new pattern should be something like:
```python
try:
    import foo
    HAS_FOO = True
except ImportError:
    HAS_FOO = False

# Additional code here

def main():
    module = AnsibleModule(...)
    if not HAS_FOO:
        module.fail_json(msg="Python foo module is required")

    # remaining code
```